### DISCUSSION TITLE
Pylance could support Cython

### DISCUSSION THREAD

**Discussion by midapsh** [microsoft/pylance-release#2678] [Enhancement]
Sometimes I stumble across packages like [Dependency Injector](https://github.com/ets-labs/python-dependency-injector) that depends on `.pxd` files. Is there any chance that you will support this extension in the future?

**Comment by tristan957** [comment]
Of note, I have just gotten done writing some Python bindings through Cython for a C library. Would have been nice to have good syntax highlighting/autocomplete, but part of me wonders if Cython shouldn't be its own extension. Cython support in the Python extension sounds out of scope, but who knows :shrug:.

**Comment by alexreg** [comment]
Any updates on this? Would be a very useful feature.

**Comment by polm** [comment]
Just a note that this seems to be a dupe of #166, which is the oldest / most active topic on Cython support.

EDIT: Ah, my mistake, this is not quite a dupe as #166 is more general, though it still is relevant for Cython support.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
